### ISSUE TITLE
Dropkick Ignores Anger [BUG]

### ISSUE THREAD

**Issue by Khunkurisu** [DU-Devs/Dragon-Universe-Public#48]
**Describe the bug**
See title.

**To Reproduce**
1. Get someone with dropkick
2. Make sure your anger is active and working.
3. Get low on hp.
4. Have other player dropkick you.

**Expected behavior**
Dropkick should proc anger for everyone involved when the normal anger conditions are met.

**Screenshots**
N/A

**Byond Details:**
- Version (ex. 512, 513, etc): any

**First identifiable version:**
Basically whatever version dropkick was added in.

**Comment by ty162** [comment]
it's probably not ignoring anger it's probably doing enough damage to 1 shot you even with anger. I know this because when I got drop kick I did anger but I had 30% health while anger

**Comment by Khunkurisu** [comment]
As mentioned above, this was not a bug.  Dropkick just has a tendency to do far more damage than is reasonable.  When an attack deals 100% hp + your current hp% as damage (or greater), it skips anger entirely.  This is almost always the case for dropkick.  Way too powerful of an attack.  Adjusted its damage to be less consistently OP.  It should be a last-ditch effort with a high risk to use, so the new damage reflects that.
